misc/pause-container/Dockerfile

```diff
@@ -15,5 +15,4 @@ FROM scratch
 MAINTAINER Amazon Web Services, Inc.
 COPY pause /usr/bin/
 WORKDIR /usr/bin
-USER 1000
```

Comment:
From the commit message https://github.com/aws/amazon-ecs-agent/pull/879/commits/b826c92f6c5c5013bfc1bf4e92e923ed93eb9805: 

This causes the agent to be started with additional capabilities as
the uids of the pause process differs from that of the agent, which
results agent needing additional capabilities to assume the netns
of the pause container